berry_fix/payload/Makefile

```diff
@@ -1,36 +1,51 @@
 TOOLCHAIN := $(DEVKITARM)
 COMPARE ?= 0
 
-ifeq ($(CC),)
-HOSTCC := gcc
-else
-HOSTCC := $(CC)
+# don't use dkP's base_tools anymore
```

Comment:
Are any of these makefile changes actually necessary? I was under the impression this was so we could build without dkP, but this is already possible now. We do it in the CI workflow

Reply:
luckytyphlosion — Today at 9:07 PM
[...]it's so people don't need to install arm-none-eabi-gcc
right now the c preprocessor used is arm-none-eabi-cpp which only comes with arm-none-eabi-gcc
so we need to use gcc -E